deno_dist/middleware/secure-headers/index.ts

```diff
@@ -1,29 +1,40 @@
+import { Buffer } from "node:buffer";
 import type { Context } from '../../context.ts'
 import type { MiddlewareHandler } from '../../types.ts'
 
+declare module '../../context.ts' {
+  interface ContextVariableMap {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
This line is not necessary.

Reply:
fixed in [1389979](https://github.com/honojs/hono/pull/2577/commits/138997996164cf4670ac7ed15876f4c7bc2aa2a5)